src/svc/invocable-scripts/paths/scripts_scriptID.yml

```diff
@@ -42,33 +82,89 @@ patch:
       schema:
         type: string
       required: true
-      description: The script ID.
+      description: |
+        Script ID.
+        The ID of the script to update.
   responses:
     '200':
-      description: The updated script.
+      description: |
+        Success.
+        The response body contains the updated script.
       content:
         application/json:
           schema:
             $ref: '../schemas/Script.yml'
+          examples:
+            $ref: '../schemas/ScriptUpdateResponse.yml'
+    '404':
+      description: |
+        Not found.
+      content:
+        application/json:
+          schema:
+            $ref: "../../../common/schemas/Error.yml"
+          examples:
+            notFound:
+              summary: |
+                The requested script was not found.
+              value: {
+                "code": "not found",
+                "message": "script \"09afa3b220fe400\" not found"
+              }
     default:
-      description: Unexpected error
+      description: Internal server error
       $ref: '../../../common/responses/ServerError.yml'
+  x-codeSamples:
+    - lang: Shell
+      label: cURL
+      source: |
+        curl -X 'PATCH' \
+          "https://cloud2.influxdata.com/api/v2/scripts/SCRIPT_ID" \
+          --header "Authorization: Token INFLUX_TOKEN" \
+          --header "Accept: application/json"
+          --header "Content-Type: application/json"
+          --data '{
+              "description": "get last point from new bucket",
+              "script": "from(bucket: updatedBucket) |> range(start: -7d) |> limit(n:1)", "language": "flux"
+            }'
 delete:
   operationId: DeleteScriptsID
   tags:
     - Invokable Scripts
   summary: Delete a script
-  description: Deletes a script and all associated records.
+  description: |
+    Deletes an invokable script and all associated records.
+
+    #### Limitations
+
+    - Only one script can be deleted per request.
+    - `scriptIDs` that do not exist return an HTTP `204` status code.
+
+    #### Related Guides
+
+    - [Invoke custom scripts](https://docs.influxdata.com/influxdb/cloud/api-guide/api-invokable-scripts/)
   parameters:
     - in: path
       name: scriptID
       schema:
         type: string
       required: true
-      description: The ID of the script to delete.
+      description: |
+        Script ID.
+        The ID of the script to delete.
   responses:
     '204':
-      description: The script is deleted.
+      description: |
+        No content.
+        The script is deleted.
```

Comment:
Might want to confirm whether this is async or sync: is the script deleted or queued for deletion?

Reply:
Looks like the delete is async, ill add clarification